src/style.css

```diff
@@ -1,3 +1,62 @@
+:root {
+  --primary-color: #000;
+  --hover-color:#00ACDC;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  margin:  0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.header-logo-img, .nav-list-link {
+  display: block;
+}
+
+.nav-list {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+}
+
+.nav-list-item:not(:first-child) {
+  margin-left: 20px;
+
+}
+
+.nav-list-link {
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: var(--primary-color);
+  height: 60px;
```

Comment:
its better not hardcode height for elements with content inside. use padding instead

Reply:
I might have misunderstood the following requirements:

- the height should be set for nav links (not the header), take it from the design.
- Nav Links should not have any padding, but have text centered

Could you please explain in more detail? thank you